add_to_app/multiple_flutters/README.md

```diff
@@ -0,0 +1,72 @@
+# multiple_flutters
```

Comment:
@xster I think this is more what you were looking for.  This consolidates most of the information without spamming the parent README.md.